content/docs/composition-vs-inheritance.md

```diff
@@ -45,11 +46,12 @@ function WelcomeDialog() {
 }
 ```
 
-**[Try it on CodePen](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
+**[Incercați in CodePen](https://codepen.io/gaearon/pen/ozqNOV?editors=0010)**
+
+Orice element din interiorul tag-ul JSX `<FancyBorder>`devine trecut în componența` FancyBorder` ca si proprietăți de `copii` Din moment ce `FancyBorder` redă` {props.children} `in interiorul unui <div> `, elementele  apar în rezultatul final.
```

Comment:
Please fix the text in quotations to be exactly as the original including spaces. If it's in quotes - `` - don't translate it.

Reply:
Like children for instance.